src/System.Runtime/tests/System/TimeZoneInfo.cs

```diff
@@ -159,15 +160,20 @@ public static void NearMinMaxDateTimeOffsetConvertTest()
         VerifyConvert(new DateTimeOffset(DateTime.MinValue.AddHours(5), new TimeSpan(-3, 0, 0)), s_strPacific, new DateTimeOffset(DateTime.MinValue, new TimeSpan(-8, 0, 0)));
 
         VerifyConvert(DateTime.MaxValue, s_strPacific, s_strSydney, DateTime.MaxValue);
-        VerifyConvert(DateTime.MaxValue.AddHours(-19), s_strPacific, s_strSydney, DateTime.MaxValue);
+        if (Interop.IsWindows) // [ActiveIssue(846, PlatformID.AnyUnix)]
```

Comment:
I've created a new issue for this:  https://github.com/dotnet/corefx/issues/2465